### ISSUE TITLE
Measurements included in sharemap with added support of annotations

### ISSUE THREAD

**Issue by filleg** [origo-map/origo#1377]
Is your feature request related to a problem? Please describe.
It would be useful if measurements was included while using the sharemap-control. 
Additionally being able to add annotations to the lines/polygons (measure-control) in form of a pop-up would also be useful.

**Comment by johnnyblasta** [comment]
Yes, measurements in shared map would be nice and has been mentioned before, but doesn't seems to have been added as a issue previously.
To add annotations should be made into a separate issue, since it is separate from this issue.

**Comment by filleg** [comment]
> Yes, measurements in shared map would be nice and has been mentioned before, but doesn't seems to have been added as a issue previously. To add annotations should be made into a separate issue, since it is separate from this issue.

Noted. Changed the issue and created a separate one [here](https://github.com/origo-map/origo/issues/1378)

**Comment by johnnyblasta** [comment]
Duplicate of #999

**Comment by johnnyblasta** [comment]
Have a solution for this now, but wait for the other PRs concerning measurement is merged before I'm sending in.
